python/xorbits/_mars/lib/nvutils.py

```diff
@@ -494,11 +494,17 @@ def get_device_status(dev_index: int) -> _nvml_device_status:
         if util_node.find("gpu_util").text == "N/A":
             gpu_util = 0
         else:
-            gpu_util = int(util_node.find("gpu_util"))
+            try:
+                gpu_util = int(util_node.find("gpu_util"))
+            except:
```

Comment:
Better catch an explicit exception and explain in what scenario this exception happen.

Reply:
Or you may print some logs when exceptions happen, which could help to learn what happened.

Reply:
Got it.